### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by iliabhz**

### CODE REVIEW

**Review by DuckySoLucky — CHANGES_REQUESTED** [review]
doesn't lgtm, didn't test

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # SkyBlock-Core
 Centralized SkyBlock Mod for Settings, Menus, Localization, Events, etc.
+
+Each mod will be it's own module on top of Skyblock-Core
```
Comment (DuckySoLucky) [inline README.md:4]: lgtm

**Comment by Ascynx** [comment]
LGTM :+1:

**Review by DuckySoLucky — CHANGES_REQUESTED** [review]
yes

**Pull request merged**